Directory.Build.targets

```diff
@@ -89,7 +89,7 @@ or
     <RemoveDir Condition="Exists('$(UnityPackageProject)')" Directories="$(UnityPackageProject)" />
   </Target>
 
-  <Target Name="CleanCocoaSDK" AfterTargets="Clean" Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'">
+  <Target Name="CleaniOSSDK" AfterTargets="Clean" Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'">
```

Comment:
We'll use this also for macOS so ideally we name it Apple or Cocoa (I think Apple might be best)

Reply:
Also, please do a find/replace in the whole repo as there are comments and instructions on contributing.md and stuff where this might need to be updated too

Reply:
I'll rename to Apple

Reply:
Actually this messes up a bit with the reusable workflow names & args... I'd have to make a conditional build when you call a sdk build for "iOS" to build "Apple".

Is it really an Apple/Cocoa build? because its output is package-dev/plugins/**iOS**

Also, it contains a `.framework` not `.xcframework` so it's a single-platform build.

Reply:
fair, we do the same as Windows vs Native where one calls the other